internal/tui/edit.go

```diff
@@ -185,7 +188,26 @@ func (m EditModel) Init() tea.Cmd {
 func (m EditModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	var cmds []tea.Cmd
 
+	if m.isForgetting {
+		switch msg := msg.(type) {
+		case tea.KeyMsg:
+			switch msg.String() {
+			case "y", "enter":
+				m.isForgetting = false
+				return m, func() tea.Msg {
+					return forgetNetworkMsg{item: m.selectedItem}
+				}
+			case "n", "esc":
+				m.isForgetting = false
+				return m, nil
+			}
+		}
```

Comment:
Let's pull this switch out into a `forgetHandler` helper function so that we can reuse it in the list view for consistent behaviour.